app/scripts/components/exploration/index.tsx

```diff
@@ -89,33 +111,93 @@ function ExplorationAndAnalysis(props: ExplorationAndAnalysisProps) {
     };
   }, [resetAnalysisController, setUrl]);
 
+  const mapPanelRef = useRef<any>(null);
```

Comment:
Have you tried using utility methods that `react-resizable-panel expose` and using one ref? https://www.npmjs.com/package/react-resizable-panels#can-a-attach-a-ref-to-the-dom-elements

Reply:
I hadn't seen this. Thank you, @hanbyul-here . that definitely helps make the code a little cleaner.

Reply:
@hanbyul-here I checked on this, and `getPanelElement` is not available in the current version of `react-resizable-panels`. will keep the changes for now, and will upgrade later.